packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -33,6 +35,7 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
   private _sliderLine = React.createRef<HTMLDivElement>();
   private _thumb = React.createRef<HTMLSpanElement>();
   private _id: string;
+  private _onKeyDownTimer = -1;
```

Comment:
I don't see this code in master right now... if it's a cherrypick shouldn't this be in master branch too (7.0)? Or is it implemented differently in 7.0?

Reply:
Strange, it's in my master. Here's a screenshot:
![Screen Shot 2019-09-09 at 6 18 32 PM](https://user-images.githubusercontent.com/13246181/64576637-56385700-d32e-11e9-9e3f-d18b4750c233.png)

Reply:
ah my mistake I think I was looking at another file